src/cli.ts

```diff
@@ -309,10 +309,14 @@ export async function run (process: NodeJS.Process): Promise<number> {
   }
 
   myConsole.log('LOG   | writing BOM to', options.outputFile)
```

Comment:
this log line should be immidiately before the `writeAllSync` happens

Reply:
Addressed in [commit](a9f21df27398c59067c627e939a1efe768b83c07).